lte/gateway/configs/mobilityd.yml

```diff
@@ -14,3 +14,6 @@
 # log_level is set in mconfig. it can be overridden here
 persist_to_redis: false
 redis_port: 6380
+
```

Comment:
is it possible to add a description of those fields? 

Also for `ipv6_session_prefix_alloc_mode` maybe add `RANDOM/HASH` as the possibilities